contracts/ContractRegistry.sol

```diff
@@ -2,15 +2,34 @@ pragma solidity 0.5.16;
 import "./spec_interfaces/IContractRegistry.sol";
 import "./IContractRegistryListener.sol";
 import "./WithClaimableRegistryManagement.sol";
+import "./spec_interfaces/ILockable.sol";
+import "./Initializable.sol";
 
-contract ContractRegistry is IContractRegistry, WithClaimableRegistryManagement {
+contract ContractRegistry is IContractRegistry, Initializable, WithClaimableRegistryManagement {
+
+	IContractRegistry previousContractRegistry;
```

Comment:
no functions of previousContractRegistry are called, better to use address previousContractRegistry to avoid assumptions on a similar interface